tests/components/recorder/test_history_db_schema_30.py

```diff
@@ -607,6 +610,14 @@ def set_state(entity_id, state, **kwargs):
             set_state(therm, 20, attributes={"current_temperature": 19.5})
         )
 
+    with patch(
```

Comment:
There were two states behind saved with the exact same microsecond which isn't actually possible in production.  Postgresql will return them in a different order so the test was failing.  I moved one of the states a microsecond later to avoid the issue.